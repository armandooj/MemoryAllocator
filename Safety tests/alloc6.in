a20
a20
p
f28
f28
f30
q