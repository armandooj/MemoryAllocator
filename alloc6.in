a20
a30
u
q