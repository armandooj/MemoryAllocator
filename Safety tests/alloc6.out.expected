ALLOC at : 4 (20 byte(s))
ALLOC at : 28 (20 byte(s))
Begin of free block list :
Free block at address 48, size 464
FREE  at : 28
Begin of free block list :
Free block at address 24, size 488
FREE  at : 28
Begin of free block list :
Free block at address 24, size 488
FREE  at : 30
Begin of free block list :
Free block at address 24, size 488